runtime/major_gc.c

```diff
@@ -77,7 +77,13 @@ uintnat caml_percent_free = Percent_free_def;
 uintnat caml_max_percent_free = Max_percent_free_def;
 uintnat caml_percent_sweep_per_mark = 120; /* TODO: benchmark this value */
 
-uintnat caml_gc_pacing_policy = 0;
+/* Allowable values of caml_gc_pacing_policy */
+
+#define GC_PACING_OCAML_53        0 /* Pacing as in OCaml 5.3 */
```

Comment:
Maybe say OCaml 5.3 + markdelay? The markdelay patch has quite a large effect on how pacing works (by changing the total work to do, rather than changing the work computation)

Reply:
I've updated the comment. I think the identifier is good enough.